vividus-extension-selenium/src/main/java/org/vividus/selenium/cloud/AbstractCloudTestStatusManager.java

```diff
@@ -78,6 +83,23 @@ public final void updateCloudTestStatus(BeforeWebDriverQuitEvent event)
         }
     }
 
+    @Subscribe
+    public final void waitUntilTestComplete(AfterWebDriverQuitEvent event)
+    {
+        waitUntilCloudTestIsCompleted(event.sessionId());
+    }
+
+    private void waitUntilCloudTestIsCompleted(String sessionId)
```

Comment:
Should I throw any exception here? I think logger.error should be enough

Reply:
log error only (make sure it appears in report)